cfme/configure/configuration/system_schedules.py

```diff
@@ -70,6 +70,11 @@ class ScheduleAddEditEntities(View):
     start_hour = BootstrapSelect("start_hour")
     start_minute = BootstrapSelect("start_min")
 
+    # After selecting action_type == automation tasks
```

Comment:
This is something that should likely be captured in a Conditional view, based on the selection of action_type. Do other action type selections control the display of the other widgets in this view?

https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget/base.py#L685

Reply:
@mshriver 
- I am agree with you. This view needs to be restructure as there are lots of widgets which are dependent on each other.
- Yes, other action_type selections control the display of the other widgets in this view